### ISSUE TITLE
hackpad.com is hardcoded in HTML file

### ISSUE THREAD

**Issue by pors** [hackpad/hackpad#70]
```
$ cat infrastructure/net.appjet.ajstdlib/streaming-iframe.html
<html>
<head>
<script type="text/javascript" src="https://hackpad.com/static/xhrXdFrame.js"></script>
</head>
<body>
</body>
</html>

```

This should come from the config var `etherpad.canonicalDomain` I think

**Comment by pors** [comment]
Addressed by commits 991fcd5d8f4740a7d4d940b55e996b25f96c6710 c8551abf24999f37b165d4116c8daa51aaa3cb85 and 90c20b1d88252dbe23e47ea1181ddf6f683ba848

@nikosprodro can you check and if all works fine close this issue?

**Comment by nikosprodro** [comment]
Also addressed by commit b6d132ba953548414bc3f191f8ae98bcb4773faf closing
